sae_lens/training/sae_trainer.py

```diff
@@ -398,8 +398,9 @@ def _run_and_log_evals(self):
                 if self.evaluator is not None
                 else {}
             )
-            for key, value in self.sae.log_histograms().items():
-                eval_metrics[key] = wandb.Histogram(value)  # type: ignore
+            if BACKEND != "swanlab":
```

Comment:
I'd rather not need to have if-statements like this in code. Can you provide an interface so swanlab works identical to wandb, and we don't need any of these checks? Or maybe make a generic `Logger` interface and have both Swanlab and WandB versions of that same interface? It shouldn't be the responsibility of the SAETrainer to keep track of what logger is being used.